### ISSUE TITLE
Attempting to run python -m langflow run produces error

### ISSUE THREAD

**Issue by radiantone** [langflow-ai/langflow#1664]
**Describe the bug**
Installing the pre-release per the README and attempting to run langflow per the README produces an error

```bash

$ python -m langflow run
...snipped.../venv/bin/python: No module named langflow.__main__; 'langflow' is a package and cannot be directly executed
```

**Browser and Version**

**To Reproduce**

**Screenshots**

**Additional context**

**Comment by ogabrielluiz** [comment]
Hey

This error might be related with the installation. Try using --force-reinstall

**Comment by tina1998612** [comment]
i also have this issue, already force reinstall

**Comment by FlorianMgs** [comment]
Same issue here. Already done --force-reinstall

**Comment by FlorianMgs** [comment]
Fixed this by installing latest pre-release in a fresh env

**Comment by InaamGPT** [comment]
I was facing the same problem but I was able to fix it with install/env of Python 3.10 and it worked fine for me.
